[2022-02-03 17:27:04,250] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_gcs_dag.download_dataset_task scheduled__2020-05-02T00:00:00+00:00 [queued]>
[2022-02-03 17:27:04,264] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_gcs_dag.download_dataset_task scheduled__2020-05-02T00:00:00+00:00 [queued]>
[2022-02-03 17:27:04,264] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 17:27:04,264] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-03 17:27:04,264] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 17:27:04,277] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2020-05-02 00:00:00+00:00
[2022-02-03 17:27:04,284] {standard_task_runner.py:52} INFO - Started process 3087 to run task
[2022-02-03 17:27:04,288] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yellow_taxi_gcs_dag', 'download_dataset_task', 'scheduled__2020-05-02T00:00:00+00:00', '--job-id', '104', '--raw', '--subdir', 'DAGS_FOLDER/yellow_taxi_gcs_dag.py', '--cfg-path', '/tmp/tmpjssc2n2f', '--error-file', '/tmp/tmp8cop8tsy']
[2022-02-03 17:27:04,290] {standard_task_runner.py:77} INFO - Job 104: Subtask download_dataset_task
[2022-02-03 17:27:04,361] {logging_mixin.py:109} INFO - Running <TaskInstance: yellow_taxi_gcs_dag.download_dataset_task scheduled__2020-05-02T00:00:00+00:00 [running]> on host 7e458922b35d
[2022-02-03 17:27:04,416] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-03 17:27:04,441] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yellow_taxi_gcs_dag
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2020-05-02T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-05-02T00:00:00+00:00
[2022-02-03 17:27:04,442] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-03 17:27:04,443] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2020-05.csv > /opt/***/output_2020-05.csv']
[2022-02-03 17:27:04,459] {subprocess.py:85} INFO - Output:
[2022-02-03 17:27:19,645] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-03 17:27:19,714] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yellow_taxi_gcs_dag, task_id=download_dataset_task, execution_date=20200502T000000, start_date=20220203T172704, end_date=20220203T172719
[2022-02-03 17:27:19,783] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-03 17:27:19,854] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-02-20 00:30:09,149] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_gcs_dag.download_dataset_task scheduled__2020-05-02T00:00:00+00:00 [queued]>
[2022-02-20 00:30:09,168] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_gcs_dag.download_dataset_task scheduled__2020-05-02T00:00:00+00:00 [queued]>
[2022-02-20 00:30:09,168] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-20 00:30:09,168] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-20 00:30:09,169] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-20 00:30:09,203] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2020-05-02 00:00:00+00:00
[2022-02-20 00:30:09,210] {standard_task_runner.py:52} INFO - Started process 2301 to run task
[2022-02-20 00:30:09,231] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yellow_taxi_gcs_dag', 'download_dataset_task', 'scheduled__2020-05-02T00:00:00+00:00', '--job-id', '478', '--raw', '--subdir', 'DAGS_FOLDER/yellow_taxi_gcs_dag.py', '--cfg-path', '/tmp/tmp_3h000f4', '--error-file', '/tmp/tmp7ipn_ki5']
[2022-02-20 00:30:09,233] {standard_task_runner.py:77} INFO - Job 478: Subtask download_dataset_task
[2022-02-20 00:30:09,359] {logging_mixin.py:109} INFO - Running <TaskInstance: yellow_taxi_gcs_dag.download_dataset_task scheduled__2020-05-02T00:00:00+00:00 [running]> on host da07aa254901
[2022-02-20 00:30:09,454] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-20 00:30:09,499] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yellow_taxi_gcs_dag
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2020-05-02T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-05-02T00:00:00+00:00
[2022-02-20 00:30:09,500] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-20 00:30:09,501] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2020-05.csv > /opt/***/output_2020-05.csv']
[2022-02-20 00:30:09,544] {subprocess.py:85} INFO - Output:
[2022-02-20 00:30:13,542] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-20 00:30:13,587] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yellow_taxi_gcs_dag, task_id=download_dataset_task, execution_date=20200502T000000, start_date=20220220T003009, end_date=20220220T003013
[2022-02-20 00:30:13,633] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-20 00:30:13,680] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
